lib/jobs/ucs-discovery.js

```diff
@@ -186,15 +199,69 @@ function UcsDiscoveryJobFactory(
         });
     };
 
+    UcsDiscoveryJob.prototype.updateRelations = function(nodeId, relations) {
```

Comment:
This is done with multiple calls to updateRelations in the createServers routine.  The first call updates the compute node, the second call updates the chassis node.